### PR TITLE
Setup cpack packaging for windows(wix) and linux (deb)

### PR DESCRIPTION
**Pull request by krocard**
- Use cpack to generate an installer for windows and .deb for linux
- Rework libxml2 cmake handling using the modern `INTERFACE_*` way.
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/01org/parameter-framework/pull/313%23issuecomment-157647710%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/313%23issuecomment-157701246%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/313%23issuecomment-158057014%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/313%23issuecomment-158069226%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/313%23issuecomment-158069473%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/313%23discussion_r45115481%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/313%23discussion_r45115492%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/313%23discussion_r45115583%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/313%23discussion_r45115621%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/313%23discussion_r45115733%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/313%23discussion_r45174278%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/313%23discussion_r45182601%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/313%23discussion_r45182906%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/313%23discussion_r45183374%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/313%23discussion_r45194327%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/313%23discussion_r45194329%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/313%23discussion_r45194332%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/313%23discussion_r45346963%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/313%23discussion_r45350530%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/313%23discussion_r45351329%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/01org/parameter-framework/pull/313%23issuecomment-157647710%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%3A-1%3A%20%22%2C%20%22created_at%22%3A%20%222015-11-18T08%3A53%3A46Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/11178583%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/tcahuzax%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%20%40dawagner%20%40tcahuzax%20All%20your%20remarks%20should%20have%20been%20answered.%22%2C%20%22created_at%22%3A%20%222015-11-18T12%3A45%3A05Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/krocard%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%20%40dawagner%20%40tcahuzax%20Be%20aware%20of%20the%20new%20patch%20%5C%22Normalize%20license%20file%20name%5C%22.%22%2C%20%22created_at%22%3A%20%222015-11-19T13%3A28%3A51Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/krocard%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3Ashipit%3A%20%22%2C%20%22created_at%22%3A%20%222015-11-19T14%3A14%3A12Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/11178583%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/tcahuzax%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3Ashipit%3A%20%22%2C%20%22created_at%22%3A%20%222015-11-19T14%3A15%3A11Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/krocard%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%202f910deed4520e9aff99086664685bfcad5f973b%20CMakeLists.txt%204%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/01org/parameter-framework/pull/313%23discussion_r45115481%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%60%23%20Add%20FindLibXml2.cmake%20in%20the%20path%60%22%2C%20%22created_at%22%3A%20%222015-11-17T20%3A26%3A09Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/krocard%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%3Ca%20href%3D%27%23crh-update-none%27%3E%3C/a%3E%22%2C%20%22created_at%22%3A%20%222015-11-18T12%3A46%3A24Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/krocard%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20CMakeLists.txt%3AL54-62%22%7D%2C%20%22Pull%202f910deed4520e9aff99086664685bfcad5f973b%20CMakeLists.txt%2013%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/01org/parameter-framework/pull/313%23discussion_r45115492%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22remove%22%2C%20%22created_at%22%3A%20%222015-11-17T20%3A26%3A20Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/krocard%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%3Ca%20href%3D%27%23crh-update-none%27%3E%3C/a%3E%22%2C%20%22created_at%22%3A%20%222015-11-18T12%3A46%3A24Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/krocard%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20CMakeLists.txt%3AL95-102%22%7D%2C%20%22Pull%202f910deed4520e9aff99086664685bfcad5f973b%20xmlserializer/CMakeLists.txt%204%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/01org/parameter-framework/pull/313%23discussion_r45115733%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22do%20not%20move%20this%22%2C%20%22created_at%22%3A%20%222015-11-17T20%3A28%3A14Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/krocard%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%20Done%22%2C%20%22created_at%22%3A%20%222015-11-18T10%3A33%3A09Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/krocard%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20xmlserializer/CMakeLists.txt%3AL37-45%22%7D%2C%20%22Pull%202f910deed4520e9aff99086664685bfcad5f973b%20cpack/CMakeLists.txt%2040%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/01org/parameter-framework/pull/313%23discussion_r45115583%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%40dawagner%20Maybe%20add%20the%20project%20email%20address%20instead%20%3F%20I%20know%20you%20created%20a%20mailing%20list%2C%20but%20could%20not%20find%20it.%22%2C%20%22created_at%22%3A%20%222015-11-17T20%3A27%3A10Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/krocard%22%7D%7D%2C%20%7B%22body%22%3A%20%22Look%20at%20the%2001.org%20project%20page.%22%2C%20%22created_at%22%3A%20%222015-11-18T09%3A03%3A40Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6430928%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/dawagner%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%20Done%22%2C%20%22created_at%22%3A%20%222015-11-18T10%3A25%3A53Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/krocard%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%3Ca%20href%3D%27%23crh-update-complete%27%3E%3C/a%3E%22%2C%20%22created_at%22%3A%20%222015-11-18T12%3A46%3A24Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/krocard%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20cpack/CMakeLists.txt%3AL1-52%22%7D%2C%20%22Pull%202f910deed4520e9aff99086664685bfcad5f973b%20cpack/CMakeLists.txt%2045%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/01org/parameter-framework/pull/313%23discussion_r45115621%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Need%20to%20install%20libxml2%20too%20for%20windows.%22%2C%20%22created_at%22%3A%20%222015-11-17T20%3A27%3A30Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/krocard%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%20done%2C%20added%20a%20TODO%22%2C%20%22created_at%22%3A%20%222015-11-18T10%3A28%3A45Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/krocard%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20cpack/CMakeLists.txt%3AL1-52%22%7D%2C%20%22Pull%20c5d37bcb91447908663a0e0786a9aec09ee1560d%20cpack/CMakeLists.txt%2054%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/01org/parameter-framework/pull/313%23discussion_r45346963%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22if%20you%20set%20the%20exact%20file%20name%20here%2C%20is%20the%20extension%20really%20needed%20%3F%22%2C%20%22created_at%22%3A%20%222015-11-19T14%3A33%3A06Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6430928%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/dawagner%22%7D%7D%2C%20%7B%22body%22%3A%20%22Because%20windows%20installer%20does%20not%20recognize%20a%20text%20file%20without%20the%20.txt%20extension.%22%2C%20%22created_at%22%3A%20%222015-11-19T15%3A00%3A46Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/krocard%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%20%22%2C%20%22created_at%22%3A%20%222015-11-19T15%3A07%3A30Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/krocard%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20cpack/CMakeLists.txt%3AL1-75%22%7D%7D%2C%20%22approved%22%3A%20%7B%22https%3A//github.com/krocard%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%7D%2C%20%22https%3A//github.com/tcahuzax%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/11178583%3Fv%3D3%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>

<a href='https://github.com/krocard'></a><a href='https://github.com/tcahuzax'></a>

- [x] <a href='#crh-comment-Pull 2f910deed4520e9aff99086664685bfcad5f973b CMakeLists.txt 4'></a> &nbsp;<b><a href='https://github.com/01org/parameter-framework/pull/313#discussion_r45115481'>File: CMakeLists.txt:L54-62</a></b>
- <a href='https://github.com/krocard'></a> `# Add FindLibXml2.cmake in the path`
- [x] <a href='#crh-comment-Pull 2f910deed4520e9aff99086664685bfcad5f973b CMakeLists.txt 13'></a> &nbsp;<b><a href='https://github.com/01org/parameter-framework/pull/313#discussion_r45115492'>File: CMakeLists.txt:L95-102</a></b>
- <a href='https://github.com/krocard'></a> remove
- [x] <a href='#crh-comment-Pull 2f910deed4520e9aff99086664685bfcad5f973b cpack/CMakeLists.txt 40'></a> &nbsp;<b><a href='https://github.com/01org/parameter-framework/pull/313#discussion_r45115583'>File: cpack/CMakeLists.txt:L1-52</a></b>
- <a href='https://github.com/krocard'></a> @dawagner Maybe add the project email address instead ? I know you created a mailing list, but could not find it.
- <a href='https://github.com/dawagner'></a> Look at the 01.org project page.
- <a href='https://github.com/krocard'></a> :+1: Done
- [x] <a href='#crh-comment-Pull 2f910deed4520e9aff99086664685bfcad5f973b cpack/CMakeLists.txt 45'></a> &nbsp;<b><a href='https://github.com/01org/parameter-framework/pull/313#discussion_r45115621'>File: cpack/CMakeLists.txt:L1-52</a></b>
- <a href='https://github.com/krocard'></a> Need to install libxml2 too for windows.
- <a href='https://github.com/krocard'></a> :+1: done, added a TODO
- [x] <a href='#crh-comment-Pull 2f910deed4520e9aff99086664685bfcad5f973b xmlserializer/CMakeLists.txt 4'></a> &nbsp;<b><a href='https://github.com/01org/parameter-framework/pull/313#discussion_r45115733'>File: xmlserializer/CMakeLists.txt:L37-45</a></b>
- <a href='https://github.com/krocard'></a> do not move this
- <a href='https://github.com/krocard'></a> :+1: Done
- [x] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/01org/parameter-framework/pull/313#issuecomment-157647710'>General Comment</a></b>
- <a href='https://github.com/tcahuzax'></a> :-1:
- <a href='https://github.com/krocard'></a> :+1: @dawagner @tcahuzax All your remarks should have been answered.
- <a href='https://github.com/krocard'></a> :+1: @dawagner @tcahuzax Be aware of the new patch "Normalize license file name".
- [x] <a href='#crh-comment-Pull c5d37bcb91447908663a0e0786a9aec09ee1560d cpack/CMakeLists.txt 54'></a> &nbsp;<b><a href='https://github.com/01org/parameter-framework/pull/313#discussion_r45346963'>File: cpack/CMakeLists.txt:L1-75</a></b>
- <a href='https://github.com/dawagner'></a> if you set the exact file name here, is the extension really needed ?
- <a href='https://github.com/krocard'></a> Because windows installer does not recognize a text file without the .txt extension.

<a href='https://www.codereviewhub.com/01org/parameter-framework/pull/313?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/01org/parameter-framework/pull/313?approve=1'></a>&nbsp;<a href='https://www.codereviewhub.com/01org/parameter-framework/pull/313?approve=0'></a>&nbsp;<a href='https://github.com/01org/parameter-framework/pull/313'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by dawagner** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/01org/parameter-framework/pull/313'>Refresh</a>.<a href='#crh-update-none'></a>

File: CMakeLists.txt [hunk 1]
```diff
@@ -54,6 +54,8 @@ execute_process(COMMAND git describe --tags --dirty
     WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}
     OUTPUT_VARIABLE PARAMETER_FRAMEWORK_VERSION OUTPUT_STRIP_TRAILING_WHITESPACE)
 
+set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}")
```
Comment (krocard) [inline CMakeLists.txt]: Add FindLibXml2.cmake in the path

File: CMakeLists.txt [hunk 2]
```diff
@@ -93,6 +95,7 @@ if(COVERAGE)
     set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -fprofile-arcs -ftest-coverage")
 endif()
 
+
```
Comment (krocard) [inline CMakeLists.txt]: remove

File: cpack/CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+# Those version numbers are extracted from git describe
+set(REGEX "^v([0-9]+)\\.([0-9]+)\\.([0-9]+).*$")
+string(REGEX REPLACE "${REGEX}" "\\1" CPACK_PACKAGE_VERSION_MAJOR "${PARAMETER_FRAMEWORK_VERSION}")
+string(REGEX REPLACE "${REGEX}" "\\2" CPACK_PACKAGE_VERSION_MINOR "${PARAMETER_FRAMEWORK_VERSION}")
+string(REGEX REPLACE "${REGEX}" "\\3" CPACK_PACKAGE_VERSION_PATCH "${PARAMETER_FRAMEWORK_VERSION}")
+
+set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Plugin-based and rule-based framework for describing and controlling parameters.")
+set(CPACK_PACKAGE_DESCRIPTION_FILE "${PROJECT_SOURCE_DIR}/README.md")
+set(CPACK_PACKAGE_VENDOR "Intel")
+set(CPACK_PACKAGE_CONTACT "David Wagner <david.wagner@intel.com>")
```
Comment (krocard) [inline cpack/CMakeLists.txt]: @dawagner Maybe add the project email address instead ? I know you created a mailing list, but could not find it.

File: cpack/CMakeLists.txt [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+set(CPACK_DEBIAN_PACKAGE_HOMEPAGE "https://01.org/parameter-framework")
+
+if (WIN32)
+    # On windows, pack compiler provided libraries (MSVC redistributable) with the project
+    include(InstallRequiredSystemLibraries)
```
Comment (krocard) [inline cpack/CMakeLists.txt]: Need to install libxml2 too.

File: xmlserializer/CMakeLists.txt [hunk 5]
```diff
@@ -37,6 +37,8 @@ add_library(xmlserializer STATIC
 
 set_target_properties(xmlserializer PROPERTIES POSITION_INDEPENDENT_CODE TRUE)
 
+target_include_directories(xmlserializer PUBLIC .)
```
Comment (krocard) [inline xmlserializer/CMakeLists.txt]: do not move this

**Comment by tcahuzax** [comment]
:-1:

File: cpack/CMakeLists.txt (hunk 3 above)
Comment (dawagner) [inline cpack/CMakeLists.txt]: Look at the 01.org project page.

Comment (krocard) [inline cpack/CMakeLists.txt]: Done

File: cpack/CMakeLists.txt (hunk 4 above)
Comment (krocard) [inline cpack/CMakeLists.txt]: :+1: done

File: xmlserializer/CMakeLists.txt (hunk 5 above)
Comment (krocard) [inline xmlserializer/CMakeLists.txt]: :+1: Done

**Comment by krocard** [comment]
:+1: @dawagner @tcahuzax All your remarks should have been answered.

**Comment by krocard** [comment]
:+1:<a href='#crh-update-none'></a>

File: CMakeLists.txt (hunk 2 above)
Comment (krocard) [inline CMakeLists.txt]: :+1:<a href='#crh-update-none'></a>

File: CMakeLists.txt (hunk 1 above)
Comment (krocard) [inline CMakeLists.txt]: :+1:<a href='#crh-update-none'></a>

File: cpack/CMakeLists.txt (hunk 3 above)
Comment (krocard) [inline cpack/CMakeLists.txt]: :+1:<a href='#crh-update-complete'></a>

**Comment by krocard** [comment]
:+1: @dawagner @tcahuzax Be aware of the new patch "Normalize license file name".

**Comment by tcahuzax** [comment]
:shipit:

**Comment by krocard** [comment]
:shipit:

File: cpack/CMakeLists.txt [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+
+# Default version does not have the TWEAK version, override it.
+set(CPACK_PACKAGE_VERSION "${CPACK_PACKAGE_VERSION_MAJOR}.${CPACK_PACKAGE_VERSION_MINOR}.${CPACK_PACKAGE_VERSION_PATCH}.${CPACK_PACKAGE_VERSION_TWEAK}")
+
+set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Plugin-based and rule-based framework for describing and controlling parameters.")
+set(CPACK_PACKAGE_DESCRIPTION_FILE "${PROJECT_SOURCE_DIR}/README.md")
+set(CPACK_PACKAGE_FILE_README "${PROJECT_SOURCE_DIR}/README.md")
+set(CPACK_PACKAGE_VENDOR "Intel")
+set(CPACK_PACKAGE_CONTACT "parameter-framework@lists.01.org")
+set(CPACK_RESOURCE_FILE_LICENSE "${PROJECT_SOURCE_DIR}/COPYING.txt")
```
Comment (dawagner) [inline cpack/CMakeLists.txt]: if you set the exact file name here, is the extension really needed ?

Comment (krocard) [inline cpack/CMakeLists.txt]: Because windows installer does not recognize a text file without the .txt extension.

Comment (krocard) [inline cpack/CMakeLists.txt]: :+1:

**Pull request merged**